addons/project/data/project_data.xml

```diff
@@ -122,143 +122,162 @@
             <field name="project_id" ref="project.project_project_data"/>
             <field name="active" eval="False"/>
             <field name="name">Welcome to Odoo</field>
-            <field name="description">Welcome! This project has the objective to show you all the main feature in the project app. Each card will help you to manage your projects easily in a few minutes.</field>
+            <field name="description"><![CDATA[<p>Welcome!</p>
```

Comment:
We use type="xml" instead of the CDATA beautiful soup. ;)

Don't worry, I'll apply it myself on your branch.

Reply:
That's good to know :-)

Thanks!